test/inductor/test_torchinductor.py

```diff
@@ -12311,19 +12311,27 @@ def func(a, b, c, d):
                 self.assertEqual(fp32_cast_in_code, upcast_to_fp32)
 
         @requires_gpu()
+        @parametrize("op", [torch.rsqrt, torch.sqrt, torch.isnan, torch.floor, torch.ceil, torch.tan, torch.atan, torch.atanh, torch.sigmoid, torch.log2, torch.log10, torch.cosh, torch.sinh, torch.acosh, torch.asinh, torch.asin, torch.acos, torch.asinh, torch.erf, torch.lgamma, torch.sin, torch.cos, torch.exp, torch.expm1, torch.exp2, torch.abs])
```

Comment:
Can we use the opinfo db and select all pointwise ops or something programatically? I am worried this list will not be updated in the future as we add more ops.

Reply:
Just iterate through the class attributes directly?

Reply:
Do you mean we should iterate through all the methods of `TritonOverrides`? That could work but this test wouldn't necessarily apply to all of them. Wouldn't we need some extra info to figure out which ones generate `libdevice` calls to a unary op? It might be possible to get this with `inspect`.